tests/phpunit/CRM/Core/Payment/AuthorizeNetTest.php

```diff
@@ -165,10 +165,10 @@ public function testCreateSingleNowDated() {
     );
 
     // turn verifySSL off
-    CRM_Core_BAO_Setting::setItem('0', CRM_Core_BAO_Setting::SYSTEM_PREFERENCES_NAME, 'verifySSL');
+    Civi::settings()->set('verifySSL', '0');
     $this->doPayment($params);
     // turn verifySSL on
-    CRM_Core_BAO_Setting::setItem('0', CRM_Core_BAO_Setting::SYSTEM_PREFERENCES_NAME, 'verifySSL');
+    Civi::settings()->set('verifySSL', '0');
```

Comment:
@eileenmcnaughton ever notice that this code is doing the opposite of what the comment says it's doing?